sms/constructor.tentative.https.any.js

```diff
@@ -0,0 +1,60 @@
+// META: title=SMS Receiver API: Constructor
+
+'use strict';
+
+promise_test(async t => {
+  let used = false;
+
+  new SMSReceiver({
+    get timeout() {
+      used = true;
+      return 60;
+    }
+  });
+
+  assert_true(used, 'constructor options "timeout" member was used');
+}, 'constructor uses timeout property');
+
+promise_test(async t => {
+  try {
+    new SMSReceiver({timeout: 0});
+    assert_unreached('Timeout 0 should reject');
+  } catch (error) {
+    assert_equals(error.name, 'TypeError');
```

Comment:
[The `assert_throws` utility function](https://web-platform-tests.org/writing-tests/testharness-api.html) is more concise and produces more descriptive error messages.